app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -4,7 +4,7 @@ module Authentication
   module AuthnJwt
     # Generic JWT authenticator that receive JWT vendor configuration and uses to validate that the authentication
     # request is valid, and return conjur authn token accordingly
-    Authenticate = CommandClass.new(
+    Authenticator = CommandClass.new(
```

Comment:
if you don't have an `Authenticator` class then you don't need to change this class name from `Authenticate`. am i missing something?

Reply:
Youre right, there is no need to change the name but I just wanted it to match the other authenticator classes